pkg/volume/util/quota/common/quota_linux_common.go

```diff
@@ -0,0 +1,105 @@
+// +build linux
+
```

Comment:
nit: `pkg/volume/util/quota/` seems like a very generic name. Worth being more specific?

Reply:
It could potentially be fsquota or the like, if you feel strongly about it.

Reply:
Slight concern about confusing readers. But don't feel strongly. We could rename in the future if there is a collision.